components/core/src/package/install.rs

```diff
@@ -299,13 +299,15 @@ impl PackageInstall {
         let mut deps: Vec<PackageIdent> = vec![];
         match self.read_metafile(file) {
             Ok(body) => {
-                let ids: Vec<String> = body.split("\n").map(|d| d.to_string()).collect();
-                for id in &ids {
-                    let package = try!(PackageIdent::from_str(id));
-                    if !package.fully_qualified() {
-                        return Err(Error::InvalidPackageIdent(package.to_string()));
+                if body.len() > 0 {
```

Comment:
Ahhh. good to know. Thats what I get for hand rolling a package. I thought we always have a DEPS file. I have no idea why I assumed that.